### PR TITLE
Update circleci image to use new cimg namespace with Go1.17

### PR DESCRIPTION
**Pull request by nywilken**

### CODE REVIEW

File: filelock/filelock.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build !solaris
 // +build !solaris
```
Comment (azr) [inline filelock/filelock.go:2]: Should we only keep the newer//more readable version here ?

Comment (azr) [reply filelock/filelock.go:2]: Maybe not in case this gets built with go 1.16

Comment (nywilken) [reply filelock/filelock.go:2]: Oh interesting. I thought it was a direct replace of the build tag. Packer currently has both so maybe we can keep it consistent, and remove once 1.18 is out. What do you think?

Comment (nywilken) [reply filelock/filelock.go:2]: I'm merging as is for now. During our sync tomorrow we can make a call to add or remove here and in Packer if we don't want to wait until Go 1.18.

Comment (azr) [reply filelock/filelock.go:2]: yup sounds good to me !

**Comment by azr** [comment]
Nice one !

**Pull request merged**